### PR TITLE
Admin Test Delete Material

### PR DESCRIPTION
**Pull request by vhhieu95**
Admin Test Delete Material

### CODE REVIEW

File: tests/Browser/Pages/Materials/DeleteMaterialTest.php [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            DB::table('materials')->delete(2);
+            $browser->click('.table tbody tr:nth-child(2) td:nth-child(7) .fa-trash')
+                ->waitFor(null, '1')
+                ->assertSee('Delete Material')
+                ->assertSee('Are you sure delete Material?')
+                ->click('#btn-modal-submit')
+                ->assertSee('Material Not Found');
+        });
+    }
+
```
Comment (ghost) [inline tests/Browser/Pages/Materials/DeleteMaterialTest.php:77]: Please add case user press **Cancel** button.

**Review by ghost — COMMENTED** [review]
Please check my comment

**Comment by vhhieu95** [comment]
I update, please check again. Thank you so much

**Comment by ghost** [comment]
LGTM 👀

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**